src/experiment/launch_experiment.jl

```diff
@@ -38,11 +54,15 @@ function launch_experiment!(
         evaluator::Union{Nothing, AbstractEvaluator}=SameInstancesEvaluator(valueSelectionArray,generator),
         restartPerInstances::Int64,
         rngTraining::AbstractRNG,
+        training_timeout =nothing::Union{Nothing, Int},
+        eval_every =nothing::Union{Nothing, Int},
+        logger = logger,
+        nbTrainingPoints = 1000,
+        device = cpu
     ) where{T <: ValueSelection, S1,S2 <: SearchStrategy}
-
-
+    
+    #inputBuffer = monitorInput()
```

Comment:
This can be uncommented to allow users to manually stop the training and launch a final evaluation.